common/text/arial-labels.ts

```diff
@@ -0,0 +1,5 @@
+export const expandedViewImageButton = 'View expanded image';
+export const searchFilterCheckBox = (label: string): string => {
+  return `Radio checkbox ${label}`;
+};
+export const searchFormInput = 'search input field';
```

Comment:
I think - from looking around, this is what is presented to a person, which should probably be "Search the catalogue" or "Search for images"